packages/jest-circus/src/legacy-code-todo-rewrite/jestAdapterInit.ts

```diff
@@ -34,9 +34,8 @@ import globals from '..';
 
 type Process = NodeJS.Process;
 
-// TODO: hard to type
```

Comment:
oh? 😛 the eslint ignore seems to still be here

Reply:
oh, sorry I will return this `TODO:` back, probably it was by accident 😊